2019.csv

```diff
@@ -8,8 +8,9 @@ PyCon APAC,2019-02-23,2019-02-24,"Manila, Philippines",PHL,,,,https://pycon-tick
 DjangoCon Europe,2019-04-10,2019-04-12,"Copenhagen, Denmark",DNK,,,,https://2019.djangocon.eu/,,
 Pycon Italia,2019-05-02,2019-05-05,"Florence, Tuscany, Italy",ITA,Grand Hotel Mediterraneo,,2019-01-06,https://www.pycon.it/en,https://www.pycon.it/en/call-for-proposals/,https://www.pycon.it/en/sponsor/
 PyCon North America,2019-05-01,2019-05-09,"Cleveland, Ohio, United States of America",USA,Huntington Convention Center,2018-11-26,2019-01-03,https://us.pycon.org/2019,https://us.pycon.org/2019/speaking/,https://us.pycon.org/2019/sponsors/prospectus/
+PyCon CZ,2019-06-14,2019-06-16,"Ostrava, Czech Republic",CZE,"Dolní oblast Vítkovice",,,https://cz.pycon.org/2019/,,https://cz.pycon.org/2019/static/pyconcz-2019-sponsorship-prospectus.pdf
 GeoPython,2019-06-24,2019-06-26,"Basel, Switzerland",CHE,,,,http://2019.geopython.net,,http://2019.geopython.net/GeoPython_Sponsoring_2019_lowres.pdf
 PyCon AU,2019-08-02,2019-08-06,"Sydney, New South Wales, Australia",AUS,International Convention Centre,,,https://2019.pycon-au.org/,,https://2019.pycon-au.org/sponsor/
 PyColorado,2019-09-07,2019-09-08,"Denver, Colorado, United States of America",USA,The Studio Loft at Ellie Caulkins Opera House,,,https://pycolorado.org,,
 DjangoCon US,2019-09-15,2019-09-19,"TBD (San Diego, CA), United States of America",USA,,,,https://djangocon.us,,
-PyCon DE 2019,2019-10-09,2019-10-13,"Berlin, Germany",DEU,,,,https://de.pycon.org/,,
\ No newline at end of file
+PyCon DE 2019,2019-10-09,2019-10-13,"Berlin, Germany",DEU,,,,https://de.pycon.org/,,

```

Comment:
Your editor is broken: it adds EOL here, which is not related to this PR.

Reply:
I don't know many people having their editors setup to remove EOLs. From my experience they usually automatically add them or leave them intact, but YMMV, it might be just my social bubble.

Reply:
I think eol or no eol, it is fine, as long as the csv is valid :)

Reply:
yeah, I used gh ui as well :)

Reply:
this shouldn't be a blocker anyway